editor/src/core/shared/project-file-types.ts

```diff
@@ -14,9 +14,15 @@ import { arrayEquals, objectEquals } from './utils'
 export type id = string
 
 export type ScenePath = {
+  type: 'scenepath'
+  sceneElementPaths: ElementPath[]
+}
+
+export type StaticScenePath = {
   type: 'scenepath'
```

Comment:
for extra super duper type safety, we could name it staticscenepath too

Reply:
We could but I'm not going to make that change as part of this as it requires even more refactoring